types/superagent/superagent-tests.ts

```diff
@@ -219,7 +219,7 @@ request.get("http://example.com/search").retry(2, callback).end(callback);
 })();
 
 // Attaching files
-const blob: Blob = new File([], "thor.png");
+const blob = {} as unknown as Blob;
```

Comment:
AFAICT, this Blob is treated by `@types/node` as equal to the one provided by lib-dom, but actually is not.  It's [`buffer.Blob`](https://nodejs.org/api/buffer.html#class-blob) which is not actually the same thing.  Per that link, it does have a constructor, so instead of faking it out with type casting, you could try writing

````ts
import type { Blob } from "buffer";
const blob = new Blob([], "thor.png");
````

where the `buffer` module is defined in `@types/node`.  I haven't checked out your fork to try this but I *think* it would work.

Reply:
On second thought, if this test is getting executed in the browser, you don't want to import anything from `buffer`.  You're still casting to Node's `buffer.Blob`, though.  The least-worst option might be just typing `blob` as `any`?

Reply:
Because of this line: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/877c54dd9bbc12e59a78d100eace738b7e7811b4/types/node/buffer.d.ts#L164

Reply:
If I modify my local `buffer.d.ts` to:

```ts
  stream(): any;
```

Then the `superagent` build is fine, but obviously that's not really viable.

Reply:
I'm not really sure it's the job of these type tests to know how to construct a `Blob`?